runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/bank/Rs2Bank.java

```diff
@@ -1296,6 +1296,38 @@ public static BankLocation getNearestBank() {
         }
         return nearest;
     }
+    /**
+     * Get the nearest bank to world point
+     *
+     * @return BankLocation
+     */
+
+    public static BankLocation getNearestBank(WorldPoint worldPoint) {
```

Comment:
having the same method with a different signature and different implementation is confusing. I would add the code from getNearestBank without param to yours. and call getNearestBank(WorldPoint worldpoint) inside getNearestBank() 

The param you pass to getNearestBank is Rs2Player.getWorldLocation

Reply:
Done refactoring the method